layouts/default.vue

```diff
@@ -14,6 +14,9 @@ import SiteFooter from "@/components/SiteFooter.vue";
 export default {
   components: { SiteFooter },
   async mounted() {
+    // eslint-disable-next-line
```

Comment:
For now, explicitly put this as a console log for us to see which env we are connected to. Can change this later if we want.

Reply:
You know, I think we should leave it, but perhaps surround it with checking for `FIREBASE_ENV != "PROD"` or somesuch. For those of us who do have prod DB access it'd be really nice to make sure we're in test mode here going forward

Reply:
not a bad idea to add the check. But for first merge, I was to explicitly see PROD to ensure it's working :)

Reply:
Sounds perfect.